pkg/controller/add_controllers.go

```diff
@@ -4,12 +4,27 @@
 package controller
 
 import (
+	"os"
+	"strings"
+
 	"github.com/open-cluster-management/governance-policy-propagator/pkg/controller/automation"
 	"github.com/open-cluster-management/governance-policy-propagator/pkg/controller/policymetrics"
 	"github.com/open-cluster-management/governance-policy-propagator/pkg/controller/propagator"
 )
 
+// reportMetrics returns a bool on whether to report GRC metrics from the propagator
+func reportMetrics() bool {
+	metrics, found := os.LookupEnv("REPORT_METRICS")
```

Comment:
Maybe DISABLE_REPORT_METRICS might help indicate that they are on by default?

Reply:
sure, makes sense to me